lib/vm/src/vmcontext.rs

```diff
@@ -866,9 +885,25 @@ impl VMBuiltinFunctionIndex {
     pub const fn get_raise_trap_index() -> Self {
         Self(13)
     }
+    /// Returns an index for wasm's `table.size` instruction for local tables.
+    pub const fn get_table_size_index() -> Self {
+        Self(14)
+    }
+    /// Returns an index for wasm's `table.size` instruction for imported tables.
+    pub const fn get_imported_table_size_index() -> Self {
+        Self(15)
+    }
+    /// Returns an index for wasm's `table.grow` instruction for local tables.
+    pub const fn get_table_grow_index() -> Self {
+        Self(16)
+    }
+    /// Returns an index for wasm's `table.grow` instruction for imported tables.
+    pub const fn get_imported_table_grow_index() -> Self {
+        Self(17)
+    }
     /// Returns the total number of builtin functions.
     pub const fn builtin_functions_total_number() -> u32 {
-        14
+        18
```

Comment:
I'm curious: perhaps there is a way to automate this in safe rust? (so we don't need to do it manually each time we add fields)

Reply:
Yeah, I briefly thought about that too. We can definitely do it with a macro, in a number of ways. I think Nick may have removed this code though or maybe the PR is still open. I believe this is the thing he was considering dropping in favor of the other, very similar thing